src/node_crypto_bio.cc

```diff
@@ -24,6 +24,7 @@
 #include "util-inl.h"
 #include <climits>
 #include <cstring>
+#include <iostream>
```

Comment:
Why this addition?

Reply:
I am doing some tinkering with this file and forget to remove it. I will fix it this evening.